packages/medusa/package.json

```diff
@@ -68,6 +68,7 @@
     "iso8601-duration": "^1.3.0",
     "jsonwebtoken": "^8.5.1",
     "medusa-core-utils": "^1.1.36",
+    "medusa-telemetry": "^0.0.15",
```

Comment:
question: Did you experience issues without this dependency?

Reply:
Yes, the medusa package was sometimes failing to build because the dependency was not explicitly declared.